### DISCUSSION TITLE
Get generated strategy output

### DISCUSSION THREAD

**Discussion by mazany** [DigiLive/mushroom-strategy#89] [Q&A] [answered]
Hi. I really like this strategy as a quick showcase, but I would like to look at the generated YAML configuration to use it in my manual dashboards.

Is that possible somehow?

Thank you.

**Comment by DigiLive** [comment]
Actually we don't generate any yaml on which the dashboard is created.
We configure the dashboard dynamically.

With some trickery it should be kind of possible, but I can't remember how I did it before.
I'll have to play a bit to see if I can find out again how to get yaml.

Once I'm getting there, I'll place another comment.

**Reply by AlexHallberg** [reply]
Hi, Did you ever find the solution to this?? Or is it just not possible?

**Comment by DigiLive** [comment]
It has something to do with a misconfiguration of yaml file, but I don't recall how I did it before.

I guess we can dump the generated dashboard info into the console or maybe a card or something.

**Comment by DigiLive** [comment]
https://github.com/AalianKhan/mushroom-strategy/issues/133#issuecomment-2042950365

**Comment by DigiLive** [comment, marked as answer]
After setting up the dashboard:

1. Click the pencil-icon at the top-right of the page.
   (The `raw configuration editor opens.)
3. Delete everything, except for the first two lines.
4. Click the x-icon at the top-left of the page.
5. Click the pencil-icon at the top-right of the page.
   (The `Dashboard editor` opens.)
6. Click the 3-dots at the top-right of the `Dashboard editor`.
7. Click `Take control`.
8. Click `Take control`.
9. Click `Done` at the top-right of the page.

The dashboard can now be edited as any other dashboard.
The yaml is available by following these steps:

1. Click the pencil-icon at the top-right of the page.
2. Click the 3-dots at the top-right of the page.
3. Click `Raw configuration editor`.

> [!NOTE]
> I have no idea how well the dashboard will work after applying these steps.
